src/sampling.jl

```diff
@@ -593,27 +741,34 @@ Reference: Efraimidis, P. S., Spirakis, P. G. "Weighted random sampling with a r
 Noting `k=length(x)` and `n=length(a)`, this algorithm takes ``O(n + k \\log k)``
 processing time to draw ``k`` elements. It consumes ``n`` random numbers.
 """
-function efraimidis_a_wsample_norep!(rng::AbstractRNG, a::AbstractArray,
-                                     wv::AbstractWeights, x::AbstractArray)
+function efraimidis_a_wsample_norep!(
+    rng::AbstractRNG,
+    a::AbstractArray,
+    wv::AbstractWeights,
+    x::AbstractArray,
+)
     n = length(a)
     length(wv) == n || throw(DimensionMismatch("a and wv must be of same length (got $n and $(length(wv)))."))
     k = length(x)
 
     # calculate keys for all items
     keys = randexp(rng, n)
-    for i in 1:n
-        @inbounds keys[i] = wv.values[i]/keys[i]
+    for i = 1:n
```

Comment:
Why change one acceptable form into another? That only creates friction.

Reply:
Sorry. Again Format Code option.